internal/crdao/services/event_tracking/track_value_updated.go

```diff
@@ -1,9 +1,9 @@
 package event_tracking
 
 import (
-	"casper-dao-middleware/internal/crdao/dao_event_parser/events"
 	"casper-dao-middleware/internal/crdao/di"
 	"casper-dao-middleware/internal/crdao/entities"
+	"casper-dao-middleware/internal/crdao/events"
```

Comment:
As mentioned above let's rename it to something more specific like `TrackSettingValueUpdated` or something.

Second, shouldn't it be migrated to CES?